apps/api/src/security/sql-sanitizer.ts

```
@@ -0,0 +1,460 @@
+  private static containsDangerousPatterns(input: string): boolean {
+    return this.DANGEROUS_PATTERNS.some(pattern => pattern.test(input));
+  }
+
+  private static buildSanitizedQuery(operation: SQLOperation): string {
+    // This is a simplified example - in production, use a proper query builder
+    const { type, table, columns, conditions } = operation;
+
+    switch (type) {
+      case 'SELECT':
+        const selectColumns = columns && columns.length > 0 ? columns.join(', ') : '*';
+        let query = `SELECT ${selectColumns} FROM ${table}`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)